Gems/EMotionFX/Code/EMotionFX/Source/BlendTreeLookAtNode.cpp

```diff
@@ -433,12 +433,12 @@ namespace EMotionFX
             ->Attribute(AZ::Edit::Attributes::ChangeNotify, AZ::Edit::PropertyRefreshLevels::EntireTree)
             ->DataElement(AZ::Edit::UIHandlers::Default, &BlendTreeLookAtNode::m_limitMin, "Yaw/pitch min", "The minimum rotational yaw and pitch angle limits, in degrees.")
             ->Attribute(AZ::Edit::Attributes::Visibility, &BlendTreeLookAtNode::GetLimitWidgetsVisibility)
-            ->Attribute(AZ::Edit::Attributes::Min, AZ::Vector2(-90.0f, -90.0f))
-            ->Attribute(AZ::Edit::Attributes::Max, AZ::Vector2(90.0f, 90.0f))
+            ->Attribute(AZ::Edit::Attributes::Min, -90.0f)
+            ->Attribute(AZ::Edit::Attributes::Max, 90.0f)
```

Comment:
Hm... that is one way to fix it of course, but what if we need to have different values per component? Or was that never supported?

Reply:
The vector input doesn't currently support setting a min/max per component, so it wouldn't have worked like that in the past. But I am creating a follow-up task to add this support, because it should be straightforward to do, just a little too much scope to get tested/done by tomorrow.

Reply:
Here is the follow-up GHI I've created to add support for setting min/max per element with a vector:

https://github.com/o3de/o3de/issues/1973